package.json

```diff
@@ -4,13 +4,14 @@
   "version": "0.0.0",
   "main": "dist/index.html",
   "scripts": {
-    "start": "./node_modules/.bin/gulp",
-    "build": "./node_modules/.bin/gulp build",
-    "test": "./node_modules/.bin/gulp test",
-    "serve": "./node_modules/.bin/gulp serve",
-    "deploy": "./node_modules/.bin/gulp deploy"
```

Comment:
NB. In `package.json` scripts, binarys in `node_modules/.bin` are already sourced, so no need to specify this.